src/base-element.js

```diff
@@ -921,10 +921,11 @@ export class BaseElement {
    *
    * @param {function()} mutator
    * @param {Element=} opt_element
+   * @param {boolean=} skipRemeasure
    * @return {!Promise}
    */
-  mutateElement(mutator, opt_element) {
-    return this.measureMutateElement(null, mutator, opt_element);
+  mutateElement(mutator, opt_element, skipRemeasure) {
```

Comment:
You might consider a new method here for the case below. This would avoid having to pass undefined Elements.

`mutateElementSkipRemeasure(mutator, opt_element)`

Reply:
+1